indy_node/server/domain_req_handler.py

```diff
@@ -271,10 +271,20 @@ def _validate_revoc_reg_def(self, req: Request):
         assert revoc_def_type
         tags = cred_def_id.split(":")
 
-        self.write_req_validator.validate(req,
-                                          [AuthActionAdd(txn_type=REVOC_REG_DEF,
-                                                         field='*',
-                                                         value='*')])
+        rev_entry, _ = self._get_current_revoc_entry_and_revoc_def(req.identifier, revoc_def_type, req.reqId)
```

Comment:
RevocRegEntry isn't needed here (moreover, it's most probably doesn't exist yet).
It's sufficient to get ReovcDef only (maybe create a new method for this).